osbrain/tests/test_proxy.py

```diff
@@ -119,6 +122,41 @@ def ping(self):
         InitTimeoutProxy('foo', timeout=1.)
 
 
+def test_agent_proxy_wait_running(nsproxy):
+    """
+    Using wait_for_running on a proxy after initialization should block until
+    the agent is running or time out.
+    """
+    ap = AgentProcess('agent')
+    ap.start()
+
+    # Get "offline" proxy
+    agent = Proxy('agent')
+    Timer(.5, agent.run).start()
+
+    # Wait long enough for the agent to start running
+    proxy = Proxy('agent', timeout=1).wait_for_running()
+    assert proxy.ping() == 'pong'
+
+
+def test_agent_proxy_wait_running_timeout(nsproxy):
+    """
+    Using wait_for_running on a proxy after initialization should block until
+    the agent is running or time out.
```

Comment:
You could use backticks ( \` ) around \`wait_for_running\` in the docstring. You could also add a dot and/or parenthesis as in `.wait_for_running()`, but that only in case you prefer that style.